worker/draft.go

```diff
@@ -1742,6 +1743,10 @@ func (n *node) InitAndStartNode() {
 		}
 	}
 
+	if n.RaftContext.IsLearner && !hasPeer {
```

Comment:
For the case of restart, should we check again after making connections to nodes in ConfState? I don't know why it is required.

Reply:
I suggest we keep this here for now.